packages/core/botpress/src/botpress.js

```diff
@@ -476,7 +476,7 @@ class botpress {
     }
 
     if (cluster.isWorker) {
-      process.send({ workerStatus: 'starting' })
+      process.send && process.send({ workerStatus: 'starting' })
```

Comment:
Just for my understanding, if PP supports cluster mode but doesn't support process.send how will the processes communicate there?

Reply:
@emirotin , good question. Phusion Passenger [doesn't support](https://www.phusionpassenger.com/library/walkthroughs/basics/nodejs/fundamental_concepts.html#vs-cluster-module) cluster module and handles creating processes by itself. I'm not sure it provides ability to communicate between processes.

Reply:
They don't fail as of my tests on @digitall-it's machine.